src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java

```diff
@@ -267,21 +274,25 @@ public void validateMedicalInventoryRow(MedicalInventory inventory, List<Medical
 				movs.addAll(movBrowserManager.getMovements(medical.getCode(), null, null, null, movFrom, movTo, null, null, null, null));
 			}
 		}
-
+		// Get all the lot of the movements
+		List<Lot> lotOfMovements = movs.stream().map(m -> m.getLot()).collect(Collectors.toList());
```

Comment:
```suggestion
		List<Lot> lotOfMovements = movs.stream().map(Movement::getLot).collect(Collectors.toList());
```
Replace lambda with method reference